plugin/storage/cassandra/spanstore/dbmodel/tag_filter.go

```diff
@@ -26,6 +26,15 @@ type TagFilter interface {
 	FilterLogFields(span *model.Span, logFields model.KeyValues) model.KeyValues
 }
 
+// NewCompositeFilter returns a TagFilter that implements all of the passed filters in order.
+func NewCompositeFilter(filters ...TagFilter) TagFilter {
```

Comment:
why do we need this constructor, if we already have NewChainedTagFilter? It just adds noise.

Reply:
I thought that perhaps the functionality to optionally create a ChainedTagFilter depending on the length of the TagFilter slice would be reusable.

Removing.